### PR TITLE
VALIDATE CATMAN

### PR DESCRIPTION
**Pull request by abdullahsuleman**
# Validate [CATMAN](https://solscan.io/token/EavJDLh8cYTAnt3QDitpKGMsPL2hq1My5g9R2P6at6Lc)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/catman_solana/status/1748818867185492066?s=46&t=6atFTC3o0TOdYNWsBrPOGA
- Coingecko/ CMC URL (If available): https://coinmarketcap.com/currencies/catman/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

File: validated-tokens.csv [hunk 1]
```diff
@@ -1073,3 +1073,4 @@ BOZO,$BOZO,BoZoQQRAmYkr5iJhqo7DChAs7DPDwEZ5cv1vkYC9yzJG,5,https://arweave.net/QN
 Pax Dollar,USDP,HVbpJAQGNpkgBaYBZQBR1t7yFdvaYVp2vCQQfKKEN4tM,6,https://424565.fs1.hubspotusercontent-na1.net/hubfs/424565/USDPLOGO.png,true
 Saros,SAROS,SarosY6Vscao718M4A778z4CGtvcwcGef5M9MEH1LGL,6,https://rapid.coin98.com/Currency/saros.png,true
 MMOSH: The Stoked Token,MMOSH,FwfrwnNVLGyS8ucVjWvyoRdFDpTY8w6ACMAxJ4rqGUSS,9,https://shdw-drive.genesysgo.net/7nPP797RprCMJaSXsyoTiFvMZVQ6y1dUgobvczdWGd35/MMoshCoin.png,true
+CATMAN,CATMAN,EavJDLh8cYTAnt3QDitpKGMsPL2hq1My5g9R2P6at6Lc,9,https://bafkreieawm4zgovoxfbgkifdyiyagsieaa6cns4jb4jafxolc5z3lqysju.ipfs.nftstorage.link,true
```
Comment (hankc97) [inline validated-tokens.csv:1076]: CATMAN,$CATMAN,EavJDLh8cYTAnt3QDitpKGMsPL2hq1My5g9R2P6at6Lc,9,https://bafkreieawm4zgovoxfbgkifdyiyagsieaa6cns4jb4jafxolc5z3lqysju.ipfs.nftstorage.link,true

**Comment by hankc97** [comment]
@0xSoju2, should meet all requirements here. Thank you for helping us

File: validated-tokens.csv (hunk 1 above)
Comment (abdullahsuleman) [reply validated-tokens.csv]: @hankc97 resolved.

**Comment by DanielCherman** [comment]
I can confirm that all the provided in formation are accurate and the token is valid

**Review by shabashab — COMMENTED** [review]
I can confirm that all the provided in formation are accurate and the token is valid. @0xSoju2

**Comment by abdullahsuleman** [comment]
@0xSoju2 please review and confirm, its been 3 weeks.

**Comment by siddhere03** [comment]
@0xSoju2 Should meet all requirements here. Please Validate

**Comment by Jymfletzfur** [comment]
Cannot validate certain information

Come into Discord

**Comment by siddhere03** [comment]
@Jymfletzfur whats your discord?
And may I know what information you can't validate?

**Comment by abdullahsuleman** [comment]
> hey @siddhere03 please remove changes to pull_request_template.md. that's not for edits.

Reverted!

**Comment by siddhere03** [comment]
@9yoi it's been reverted. Is it all good now?

**Comment by 9yoi** [comment]
There's still a diff in the file if you look at files changed. Made you a fresh PR and closing this

**Comment by siddhere03** [comment]
> There's still a diff in the file if you look at files changed. Made you a fresh PR and closing this

Ok Thanks 👍

**Pull request closed without merging**